lib/aws/http_client.ex

```diff
@@ -2,24 +2,26 @@ defmodule Aws.HTTPClient do
   @moduledoc """
   Specifies the behaviour of a HTTP Client.
 
-  You can switch the default HTTP client which uses hackney underneath by defining a different implementation by setting the `:http_client` configuration in AWS.Client:
+  You can switch the default HTTP client which uses hackney underneath
+  by defining a different implementation by setting the `:http_client`
+  configuration in AWS.Client:
 
     client = %AWS.Client{http_client: {MyHttpClient, []}}
     Aws.SNS.publish(client, "My message")
```

Comment:
Please indent with four spaces since we are here. :)

Reply:
You already did 😄

Reply:
Ooops! You did it in other files. I will fix this 😃